provisioning/windows-provision.ps1

```diff
@@ -259,6 +259,11 @@ $downloads = [ordered]@{
             # Installation of make for Windows
             & "choco.exe" install make --yes --no-progress --limit-output --fail-on-error-output;
             & "choco.exe" install vagrant --yes --no-progress --limit-output --fail-on-error-output --version "${env:VAGRANT_VERSION}";
+            # install .NET 3.5 for MSI build
+            & "choco.exe" install dotnet3.5 --yes --no-progress --limit-output --fail-on-error-output
+            if(Test-Path "C:\Windows\Logs\DISM\dism.log") {
+                Get-Content "C:\Windows\Logs\DISM\dism.log"

```

Comment:
Installation  keeps failing in the context of chocolatey with the following DCIM error:

https://gist.github.com/dduportal/3a725124ab0d88184980ef72a9685892


It's weird because the Goss test responds `True`. Is there arlreay a 3.5 .NET install in the system images?

Reply:
Not sure, I guess if the goss test is getting true, then it is?!

Reply:
That is the question :) 

- I assume this test should fail on the current VMs on ci.jenkins.io (I'm double checking)
- If yes (it fails today in production) then it means that the chocolatey install in this PR does something to make the test pass. Even if the chocolatey install "fails" (kinda).
  - We could stop caring and go with it.
  - Or we could find the proper flag to pass to chocolatey to avoid the error at all (better but heavier in term of effort).
  

I'm currently focusing on the Windows in EC2 + fixing the goss test errors on Windows for `hadolint` CLI. As part of the debugs (I am building an instance with the usual packer process but I can access it to debug it in remote desktop), I'll check this.

Reply:
I confirm that, in production today ([version xxx of the image]()) , the test answers `False`. It confirms your chocolatey install does what we want (looks like).

Gotta check the DSIM log to check if the error isn't as minor "post install step" which we could ignore or disable through flag.